integration/test_helpers.go

```diff
@@ -83,6 +97,12 @@ func NewFixtures(t *testing.T) *Fixtures {
 	tmpDir, err := ioutil.TempDir("", "akash_integration_"+t.Name()+"_")
 	require.NoError(t, err)
 
+	// Prevent akashd errors on exit due to data saving behavior.
+	tmpStat, err := os.Lstat(tmpDir)
+	require.NoError(t, err)
+	err = os.MkdirAll(fmt.Sprintf("%s/.akashd/data/cs.wal", tmpDir), tmpStat.Mode())
```

Comment:
shouldn't `cs.wal` be a file?

Reply:
I saw errors trying to write beneath it as a directory, thus mkdir call.